src/Smiley.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const Smiley = styled.div`
```

Comment:
I think `SmileyDiv` is more clear here for this component.

Reply:
Makes sense.